descwl_shear_sims/galaxies.py

```diff
@@ -74,10 +75,15 @@ def make_galaxy_catalog(
             )
 
         if gal_type == 'wldeblend':
+            assert layout is not None, \
```

Comment:
In the past the default has been random, so we should also default to that here rather than making it a required argument